src/App.js

```diff
@@ -1,24 +1,51 @@
-import logo from './logo.svg';
-import './App.css';
+import { Header } from "./components/headerComponent/Header";
+import "./App.css";
+import oldmovies from "./data.json";
+import { MovieList } from "./components/movieList/MovieList";
+import { useState } from "react";
 
 function App() {
+  const [movies, setmovies] = useState(oldmovies);
+  const [movieName, setmovieName] = useState("");
+
+  function handleChange(event) {
+    setmovieName(event.target.value);
+    console.log(movieName);
+  }
+
+  function handleAdd() {
+    console.log(movies);
+    const newId = movies[movies.length - 1].id + 1;
+    const newObject = {
+      id: newId,
+      title: movieName,
+      description: "",
+      posterURL: "",
+      rating: "",
```

Comment:
The user should be able to insert the data for the movie's description, poster, and rating.